python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py

```diff
@@ -131,3 +131,43 @@ def get_scheduler(self) -> typing.Tuple[int, CoreScheduler]:
         scheduler = CoreScheduler(self.surveys, nside=self.nside)
 
         return self.nside, scheduler
+
+
+def without_keys(d: dict, keys: typing.List[str]) -> dict:
+    """Return dictionary with keys removed.
+
+    Parameters
+    ----------
+    d : `dict`
+        Input dictionary
+    keys : `list[str]`
+        List of keys to be removed.
+
+    Returns
+    -------
+        dict
+    """
+    return {x: d[x] for x in d if x not in keys}
+
+
+def get_comcam_sv_targets(exclude: typing.List[str] = []) -> dict:
+    """Load candidate targets for ComCam science observations.
+
+    Parameters
+    ----------
+    exclude : `list[str]`
+        List of target names to exclude when loading.
+
+    Returns
+    -------
+    target_dict : `dict`
+        Dictionary of candidate target names and coordinates.
+    """
+    infile = str(get_data_dir() / "field_survey_centers.yaml")
+    with open(infile) as stream:
+        try:
+            targets_dict = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

Comment:
You really want to skip parsing yaml errors? I would suggest you remove this try/except here. Or is there a reason I am missing to do this?

Reply:
Removed the try/except